app/services/common_indexers/base.rb

```diff
@@ -113,6 +114,14 @@ def respond_to_missing?(sym)
 
     private
 
+      def label_qualifier(type)
+        if type == 'nul_contributor'
```

Comment:
For ease of maintenance, I'd suggest defining an `EXPLICIT_TYPES = { 'nul_contributor' => 'Contributor }.freeze` hash constant up top and then replacing this entire `if/end` block with

```ruby
        EXPLICIT_TYPES[type] || type.capitalize
```